src/python/perfflowaspect/advice_chrome.py

```diff
@@ -124,6 +126,8 @@ class ChromeTracingAdvice:
     #             pid: process id
     # To change the directory in which the log file is created
     #     PERFFLOW_OPTIONS="log-dir=DIR"
+    # To disable logging (default: log-enable=True)
+    #     PERFFLOW_OPTIONS="log-enable=False"
```

Comment:
Could we add a simple docs page for this for users, along with a simple test case (a script that sets the enable/disable in PERFFLOW_OPTIONS)? We could extend the existing test case `t0001-pybinding-basic.t` to ensure that the options work as intended.

Reply:
Docs have been updated and additional unit tests added to test this new option.